src/chip.tsx

```diff
@@ -56,9 +58,21 @@ type ClickableChipProps = ExclusifyUnion<HrefChipProps | ToChipProps | OnPressCh
 type ChipProps = ExclusifyUnion<SimpleChipProps | ClosableChipProps | ToggleChipProps | ClickableChipProps>;
 
 const Chip: React.FC<ChipProps> = (props: ChipProps) => {
-    const {Icon, children, id, dataAttributes, active, onClose} = props;
+    const {Icon, children, id, dataAttributes, active, badge, onClose} = props;
     const {texts, isDarkMode} = useTheme();
     const overAlternative = useThemeVariant() === 'alternative';
+    const {isTabletOrSmaller} = useScreenSize();
+
+    const paddingLeft = Icon && isTabletOrSmaller ? 16 : 8 && Icon ? 8 : 20 && isTabletOrSmaller ? 20 : 12;
+    const paddingRight = isTabletOrSmaller ? 20 : 12;
+    const paddingIcon = isTabletOrSmaller ? 16 : 8;
+
+    const renderBadge = () => {
+        if (!badge) {
+            return null;
+        }
+        return <div>{badge === true ? <Badge /> : <Badge value={badge} />}</div>;
```

Comment:
This solves the alignment problems, but I dont really know if this is correct 😄 (don't listen to me a lot)

div is causing problems
`return <>{badge === true ? <Badge /> : <Badge value={badge} />}</>;`